### PR TITLE
Fix some easy typos in cmd/* :D

### PR DESCRIPTION
**Pull request by xulike666**
Signed-off-by: Aaron.L.Xu <likexu@harmonycloud.cn>

**- What I did** fix typo

**- How I did it**  code review

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: cmd/docker/docker.go [hunk 1]
```diff
@@ -5,6 +5,8 @@ import (
 	"fmt"
 	"os"
```
Comment (duglin) [inline cmd/docker/docker.go]: remove this blank line

**Comment by YuPengZTE** [comment]
LGTM

**Comment by duglin** [comment]
lgtm

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Pull request merged**